rollup.utils.js

```diff
@@ -0,0 +1,201 @@
+const fs = require("fs");
```

Comment:
This file holds utilities used by all package-specific `rollup.config.js` files, since we can't export them directly from the root `rollup.config.js` because `rollup` expects a single function export when using a CJS module :/.  We may be able to move to a `rollup.config.mjs` file and then move the utils into there but I figured keeping the surface area of this change smallest pre-merge makes sense, and we can look into the `.mjs` file after the merge if we want to get rid of the utils file.  Happy to look into it ahead of time if folks don't have concerns though.